### DISCUSSION TITLE
Sesh #2

### DISCUSSION THREAD

**Discussion by emilyyli** [LiNk-NY/IntroR#36] [Q&A] [answered]
![image](https://github.com/CUNYSPHCode/IntroR/assets/171090659/521d6ddb-933c-4952-acbd-0ede4f5d16ce)

For some reason for worksheet A06 from Session #2, I can't get my code to work. I have the same code as shown in the recorded lecture though.

**Comment by KrisDelaPena** [comment, marked as answer]
Hello! Can you try the help function again but instead of ?magritter::'%>%' using apostrophes/single quotation marks, you use backticks (`)  (its in the same key as the ~ (tilde) key on a Windows keyboard)?
